### PR TITLE
🎓 Student Lecture --- Novelty Search 

### PR DESCRIPTION
**Pull request by kaecampbell87**
### What

This is an upload of the first draft of my presentation. It contains the .rst file and a few images.

### Why

I need feedback! Also I'm not entirely sure it looks good as a website since I didn't build it locally.

### Where

The files are all new files that live in a new directory, /site/student-lectures/novelty-search

### Additional Notes

I hope you have a good break, and that I get this presentation sorted without being too much of a bother.

### CODE REVIEW

**Comment by jameshughes89** [comment]
The repo will build the website automatically, so you can actually just download the website it builds and then check it out yourself. 

[Here is the link the the "github action" where you can find the website built](https://github.com/jameshughes89/cs4XX-EvolutionaryComputation/actions/runs/6757232536)

[Here is a just the link to the download](https://github.com/jameshughes89/cs4XX-EvolutionaryComputation/suites/17910300543/artifacts/1028820762)


I have not looked at what you have yet, but I figured I would let you know how to view the website so you can do any editorial changes before I actually have a look at your content. Assuming you have not added your page to the index, you will probably have to open the file directly from the folders (the path you specified).

**Comment by jameshughes89** [comment]
I have converted the PR to a "draft". 

Once you have gone through the actual built website, let me know so I know it's ready to review.

**Comment by kaecampbell87** [comment]
I added another commit. Basically just some more images to make it less of a wall of text.

It should be ready for review and comment.

One thing I will say is that I am very unsure of the presentation time. It is likely I may wind up skipping the later parts of the 'potential problems' section when I present.

File: site/student-lectures/novelty-search/novelty search.rst [hunk 1]
```diff
@@ -0,0 +1,368 @@
...
+
+But what happens if we remove the walls?
+
+    .. figure:: simple_maze.png
+        :width: 224 px
+        :align: center
+
+        The maze without walls.
+
+If the x and y dimensions are unbounded - that is to say the potential area to explore is infinite - intuitively, there
```
Comment (jameshughes89) [inline site/student-lectures/novelty-search/novelty search.rst:229]: dash

File: site/student-lectures/novelty-search/novelty search.rst [hunk 2]
```diff
@@ -0,0 +1,368 @@
...
+The ending result of a novelty search is a collection of novel points. While hopefully the global optimum is *near* one
+of those points, it is perhaps unlikely that one of the points will achieve it exactly.
+
+    .. figure:: unoptimized.png
+        :width: 600 px
+        :align: center
+
+        A potential solution that is clearly in need of being further optimized.
+
+What the solutions provided by a novelty search provide are good *starting* points for a traditional objective based
```
Comment (jameshughes89) [inline site/student-lectures/novelty-search/novelty search.rst:251]: provided provide (reads funny)

**Review by jameshughes89 — COMMENTED** [review]
1. I see you credited some of the images, but be careful here. You can set a `:target:` for the images to link directly to the source or person too.  
2. To be clear. there is nothing wrong with your formatting, but I suggest moving using the point form strategy like I do as it make make it easier to present than the paragraphs. The way you have it may take a looong time to present/

**Comment by kaecampbell87** [comment]
I incorporated the various changes you mentioned, and added :target: tags.

The various images of the very basic maze have no image credit given, because I made them myself using some basic software.

Your point about my format taking a looong time to present is taken. I think I will stick with it instead of points though, because this way in theory you could skip my presentation and the webpage stands on its own well enough.

I hope it winds up not being boring, that'd be a shame because I really do feel like it is a neat topic. I don't really plan on just reading all that text verbatim, more like just going over it in that order and making use of the images along the way.

**Comment by kaecampbell87** [comment]
Oh shoot I think I might have been dumb and included my pso.py file in the last merge, I need to fix that

**Comment by kaecampbell87** [comment]
Okay, I think I fixed that.

**Review by jameshughes89 — APPROVED** [review]

**Comment by jameshughes89** [comment]
Add your lecture to index.

**Comment by kaecampbell87** [comment]
Done. I put it inside the student lectures toctree.

I also changed one of my file names for consistency.

**Review by jameshughes89 — APPROVED** [review]
nice

**Pull request merged**